frontend/src/views/coverage/enrollment/ResidentDetails.vue

```diff
@@ -0,0 +1,385 @@
+        <AppCol class="col4">
+          <AppInput :e-model="v$.mailingAddressCity" id="mailingAddressCity" label="City" type="text" v-model.trim="mailingAddressCity" />
+        </AppCol>
+        <AppCol class="col4">
+          <AppSelect :e-model="v$.mailingAddressProvince" id="mailingAddressProvince" label="Province" v-model="mailingAddressProvince" :options="provinceOptions" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col4">
+          <AppInput :e-model="v$.mailingAddressPostalCode" id="mailingAddressPostalCode" label="Postal Code" type="text" v-model.trim="mailingAddressPostalCode" />
+        </AppCol>
+      </AppRow>
+
+      <AppRow>
+        <AppCol class="col4">
+          <AppSelect :e-model="v$.priorResidenceCode" id="priorResidenceCode" label="Prior Residence Code" v-model="priorResidenceCode" :options="priorResidenceOptions" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col4">
+          <AppInput
+            :e-model="v$.otherProvinceHealthcareNumber"
+            id="otherProvinceHealthcareNumber"
+            label="Other Province Healthcare Number (If Applicable) (Optional)"
+            type="text"
+            v-model.trim="otherProvinceHealthcareNumber"
+          />
+        </AppCol>
+      </AppRow>
+
+      <AppRow>
+        <AppButton :disabled="searching" mode="primary" type="submit">Submit</AppButton>
+        <AppButton @click="resetForm" mode="secondary" type="button">Clear</AppButton>
+      </AppRow>
+    </form>
+  </div>
+</template>
+<script>
+import AppButton from '../../../components/AppButton.vue'
+import AppCol from '../../../components/grid/AppCol.vue'
+import AppDateInput from '../../../components/AppDateInput.vue'
+import AppInput from '../../../components/AppInput.vue'
+import AppRow from '../../../components/grid/AppRow.vue'
+import AppOutput from '../../../components/AppOutput.vue'
+import AppSelect from '../../../components/AppSelect.vue'
+import useVuelidate from '@vuelidate/core'
+import {
+  validateGroupNumber,
+  validateGroupMemberNumber,
+  validateDepartmentNumber,
+  validateTelephone,
+  VALIDATE_GROUP_NUMBER_MESSAGE,
+  VALIDATE_GROUP_MEMBER_NUMBER_MESSAGE,
+  VALIDATE_DEPARTMENT_NUMBER_MESSAGE,
+  VALIDATE_TELEPHONE_MESSAGE,
+} from '../../../util/validators'
+import { required, helpers } from '@vuelidate/validators'
+import dayjs from 'dayjs'
+import { API_DATE_FORMAT } from '../../../util/constants'
+import { formatPersonName } from '../../../util/utils'
+
+export default {
+  name: 'ResidentDetails',
+  props: {
+    resident: {
+      phn: '',
+      givenName: '',
+      secondName: '',
+      surname: '',
+      dateOfBirth: '',
+    },
+  },
+  components: {
+    AppButton,
+    AppCol,
+    AppDateInput,
+    AppInput,
+    AppOutput,
+    AppRow,
+    AppSelect,
+  },
+  setup() {
+    return {
+      v$: useVuelidate(),
+    }
+  },
+  data() {
+    return {
+      searching: false,
+      searchOk: false,
```

Comment:
Is searching the right terminology here? Could use something generic like submitting.

Reply:
Changed to more generic term submitting